solr/core/src/java/org/apache/solr/handler/IndexFetcher.java

```diff
@@ -364,8 +370,13 @@ private void fetchFileList(long gen) throws IOException {
     params.set(COMMAND, CMD_GET_FILE_LIST);
     params.set(GENERATION, String.valueOf(gen));
     params.set(CommonParams.WT, JAVABIN);
-    params.set(CommonParams.QT, ReplicationHandler.PATH);
-    QueryRequest req = new QueryRequest(params);
+    var req =
```

Comment:
[Q] SolrJ has a SolrRequest for the v2 equivalent of this API: ReplicationApi.FetchFileList. If this code is targeting 10.0 only, wdyt of using that here instead of GSR?